common/models/preview-task.js

```diff
@@ -1,3 +1,31 @@
 module.exports = function(PreviewTask) {
-
+    PreviewTask.claim = function(renderer, cb) {
+    PreviewTask.findOne({where: {renderer: renderer, startTime: null}},
```

Comment:
Consider the case where a job is started but doesn't complete -- maybe the server hits an error, or it reboots, etc. The startTime is not null, but the job should eventually be claimed again.

Could we something like "where expireTime is null or expireTime is in the past"?